packages/create/templates/dbos-drizzle/src/main.ts

```diff
@@ -91,7 +91,7 @@ app.get('/greeting/:name', (req: Request, res: Response) => {
 async function main() {
   DBOS.setConfig({
     name: 'dbos-drizzle',
-    databaseUrl: process.env.DBOS_DATABASE_URL,
+    systemDatabaseUrl: process.env.DBOS_SYSTEM_DATABASE_URL,
```

Comment:
I'm not sure about this change. All of these templates have a datasource, and we probably want to use `DBOS_DATABASE_URL` with them so they can deploy to DBOS Cloud. Maybe also want to add a comment about this specific env var like I added to the [toolbox demo app ](https://github.com/dbos-inc/dbos-demo-apps/blob/main/typescript/dbos-node-toolbox/src/main.ts#L13-L14)

Reply:
I've tested that all these templates can deploy to DBOS Cloud without `DBOS_SYSTEM_DATABASE_URL` set, because it'll fall back to use the default `DBOS_DATABASE_URL` with `_dbos_sys` suffix. It'll be set from the DBOS Cloud soon.